<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifyccdialog</class>
 <widget class="QDialog" name="modifyccdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify CC</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Modify Credit Cards</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Update Credit Card Debt:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QComboBox" name="ccSelector"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="ccUpdateAmmountDSP">
         <property name="maximum">
          <double>5999.989999999999782</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="updateCCButton">
         <property name="text">
          <string>Update CC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add New Credit Card:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QLineEdit" name="newCCLineEdit"/>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="newCCAmmontDSP">
         <property name="maximum">
          <double>5999.989999999999782</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addCCButton">
         <property name="text">
          <string>Add CC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
